dist/README

```diff
@@ -0,0 +1,26 @@
+The rkt binary and stage1 images provided in this tarball should be
+usable immediately after unpacking. rkt will use stage1-coreos.aci
+from this directory by default as a fallback. The steps below will
+make rkt more usable though (like making rkt available to everyone on
+the system, enabling rkt fetch for non-root users, integrating it with
+systemd, making rkt's --stage1-from-dir usable, etc).
+
+1. copy rkt to /usr/bin
+  - cp rkt /usr/bin
+2. add a rkt group
+  - groupadd rkt
```

Comment:
Can you add the steps mentioned in https://github.com/coreos/rkt/pull/2225/files#r54096776?